index.html

```diff
@@ -0,0 +1,584 @@
+          <p> NVIDIA GPU based data science libraries in such as cuDF (similar to Pandas), cuML (similar to
+            scikit-learn), and cuGraph (similar to networkX). Learn more about our 14+ libraries and more about
+            hardware-software requirements. </p>
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-light fa-square-code"></i> Why Use RAPIDS</h2>
+          <p> Spend less time training and moving data, more time optimizing and working. 20x - 100billion X performance
+            increase over CPU with TCOs up to 7x. RAPIDS enables End to end GPU data science, with familiar APIs and
+            libraries that seamlessly work together. Use RAPIDS with a local desktop, in the cloud, or scale out on
+            large clusters. Learn more about RAPIDS acceleration. </p>
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-light fa-square-code"></i> Open Source </h2>
+          <p>RAPIDS is an open source project, with our repositories freely available on GitHub. Being open source
+            enables a whole ecosystem of contributors, adopters, and integrators using RAPIDS to build their own
+            projects. Find out more in our ecosystems page, what RAPID enables, and what it can do for your project with
+            our example use cases. </p>
+        </div>
+      </div>
+
+      <div class="columns mt-5">
+        <div class="column is-one-third">
+          <h2><i class="fa-light fa-square-code"></i> Pandas to cuDF</h2>
+          cuDF is nearly a drop in replacement to Pandas for most use cases yet has greatly improved performance.
+
+          <canvas id="cuDFchart" width="400" height="200"></canvas>
+
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-light fa-square-code"></i> scikit-learn to cuML </h2>
+          cuML is nearly a drop in replacement to scikit-learn for most use cases yet has greatly improved performance.
+
+          <canvas id="cuMLchart" width="400" height="200"></canvas>
+
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-light fa-square-code"></i> NetworkX to cuGraph </h2>
+          cuGraph is nearly a drop in replacement to networkX for most use cases yet has greatly improved performance.
+
+          <canvas id="cuGraphChart" width="400" height="200"></canvas>
+
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <div class="top-slant-container">
+    <div class="top-slant-down has-background-rapids-purple-primary"></div>
+  </div>
+
+  <!-- Quick Start -->
+  <section class="section has-background-rapids-purple-primary has-text-white">
+    <div class="container">
+      <h1 class="mb-5 has-text-white">Quick Start</h1>
+      <div class="columns">
+        <div class="column is-half">
+          <h2 class="has-text-white"><i class="fa-light fa-square-code"></i> Quick Local Install </h2>
```

Comment:
Yes there will be several things on the docs side that will need updating in parallel. Though this would technically be under a new 'install' docs section. We should talk about where we draw the line for the deployment section.